src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasPointAndLineSlicer.java

```diff
@@ -205,195 +263,132 @@ private boolean isOutsideWorkingBound(final Map<String, String> tags)
      */
     private void processLineSlices(final Line line, final List<Geometry> slices)
     {
-        if (slices == null || slices.isEmpty())
-        {
-            // No slices generated or an error in slicing, create missing country code
-            final Map<String, String> tags = new HashMap<>();
-            tags.put(ISOCountryTag.KEY, ISOCountryTag.COUNTRY_MISSING);
-            this.slicedPointAndLineChanges.updateLineTags(line.getIdentifier(), tags);
-            updateLineShapePoints(line);
-        }
-        else if (slicesBelongToSingleCountry(slices)
-                && !getCountryBoundaryMap().shouldForceSlicing(line))
+        // Used to generate identifiers for new points and lines
+        final CountrySlicingIdentifierFactory lineIdentifierFactory = new CountrySlicingIdentifierFactory(
+                line.getIdentifier());
+        final PointIdentifierFactory pointIdentifierFactory = new PointIdentifierFactory(
+                line.getIdentifier());
```

Comment:
Is it expected they both start from the line ID?

Reply:
I made this change to simplify some logic. previously, we used the slice's ID, and limited to 999 points generated. Now, we use the original Line ID, but expand to an allowable 999,999 points. while this is technically smaller in overall point generation space, it's actually functionally larger as the previous logic would fail out if any slice ran past 999, which sometimes happens. Now, one large slice can run along a country border and use several thousand points without issue.